tedana/workflows/tedana.py

```diff
@@ -311,6 +322,23 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     n_samp, n_echos, n_vols = catd.shape
     LGR.debug('Resulting data shape: {}'.format(catd.shape))
 
+    # check if TR is 0
+    tr = ref_img.header.get_zooms()[-1]
+    if tr == 0 and user_tr == 0:
+        raise IOError(' Dataset has a TR of 0. This indicates incorrect'
+                      ' header information. Please override the TR value'
+                      ' with the --TR flag (see tedana -h for more help)'
+                      ' or fix your file header.')
+    elif user_tr != 0:
+        # Coerce TR to be user-supplied value
+        zooms = ref_img.header.get_zooms()
+        new_zooms = (zooms[0], zooms[1], zooms[2], user_tr)
+        ref_img.header.set_zooms(new_zooms)
+
+        if tr != user_tr:
+            LGR.warning('Mismatch in header TR and user-supplied TR,'
+                        ' please verify. Proceeding anyway.')
```

Comment:
It would be nice to say that it's specifically proceeding with user-supplied TR here.

Reply:
I would also make this an `info` and drop the warning elements of the message. E.g., just "Overriding header TR with user-supplied TR.").